tensorflow_datasets/core/visualization.py

```diff
@@ -28,6 +27,18 @@
 from tensorflow_datasets.core import lazy_imports_lib
 
 
+matplotlib = lazy_imports_lib.LazyImporter.matplotlib
```

Comment:
The goal of lazy import is to avoid imports in the global scope, for people not using the feature. This should be moved inside the function, otherwise it will crash for users not having installed matplotlib